src/metrics.ts

```diff
@@ -126,7 +135,7 @@ export default class Metrics extends EventEmitter {
         return true;
     }
 
-    sendMetrics(): boolean {
+    async sendMetrics(): Promise<boolean> {
```

Comment:
Making this return a Promsie would that not require the calling method to `await` it to make sure it actually is executed?  

(`this.sendMetrics();` on line 79).

Reply:
nvm; this is not a requirement i javascript